lightning/src/ln/features.rs

```diff
@@ -376,6 +385,9 @@ mod sealed {
 	define_feature!(47, SCIDPrivacy, [InitContext, NodeContext, ChannelTypeContext],
 		"Feature flags for only forwarding with SCID aliasing. Called `option_scid_alias` in the BOLTs",
 		set_scid_privacy_optional, set_scid_privacy_required, supports_scid_privacy, requires_scid_privacy);
+	define_feature!(49, PaymentMetadata, [InvoiceContext],
```

Comment:
Saw there's a module comment documenting all the features LDK supports, should this be added there? Also saw zero conf is missing there.